client/src/app/site/users/components/user-detail/user-detail.component.html

```diff
@@ -0,0 +1,152 @@
+            <fa-icon icon='ellipsis-v'></fa-icon>
+        </button>
+    </div>
+
+    <mat-menu #userExtraMenu="matMenu">
+        <button mat-menu-item class="red-warning-text" (click)='deleteUserButton()' translate>Delete User</button>
+    </mat-menu>
+
+</mat-toolbar>
+
+<mat-card class="os-card" *osPerms="'users.can_see_name'">
+    <form [ngClass]="{'mat-form-field-enabled': editUser}" [formGroup]='personalInfoForm' (ngSubmit)='saveUser()' *ngIf="user">
+        <h3 translate>Personal Data</h3>
+        <div>
+            <!-- Title -->
+            <mat-form-field class='form30 distance force-min-with'>
+                <input type='text' matInput placeholder='{{"Title" | translate}}' formControlName='title' [value]='user.title'>
+            </mat-form-field>
+
+            <!-- First name -->
+            <mat-form-field class='form30 distance force-min-with'>
+                <input type='text' matInput placeholder='{{"First Name" | translate}}' formControlName='first_name'
+                    [value]='user.firstName'>
+            </mat-form-field>
+
+            <!-- Last name -->
+            <mat-form-field class='form30  force-min-with'>
+                <input type='text' matInput placeholder='{{"Last Name" | translate}}' formControlName='last_name'
+                    [value]='user.lastName'>
+            </mat-form-field>
+        </div>
+
+        <div *osPerms="'users.can_see_extra_data'">
+            <!-- E-Mail -->
+            <mat-form-field class='form100'>
+                <input type='email' matInput placeholder='{{"EMail" | translate}}' name="email" formControlName='email'
+                    [value]='user.email'>
+                <mat-error *ngIf="personalInfoForm.get('email').hasError('email')">
+                    Please enter a valid email address
+                </mat-error>
+            </mat-form-field>
+        </div>
+
+        <div>
+            <!-- Strcuture Level -->
+            <mat-form-field class='form70 distance'>
+                <input type='text' matInput placeholder='{{"Structure Level" | translate}}' formControlName='structure_level'
+                    [value]='user.structureLevel'>
+            </mat-form-field>
+
+            <!-- Partizipant Number -->
+            <mat-form-field class='form20 force-min-with'>
+                <input type='number' matInput placeholder='{{"Participant Number" | translate}}' formControlName='number'
+                    [value]='user.participantNumber'>
+            </mat-form-field>
+        </div>
+
+        <div>
+            <!-- Groups -->
+            <mat-form-field class='form100'>
+                <mat-select placeholder='{{"Groups" | translate}}' formControlName='groups_id' multiple>
+                    <mat-option *ngFor="let group of groups" [value]="group.id">{{group}}</mat-option>
+                </mat-select>
+            </mat-form-field>
+        </div>
+
+        <div>
+            <!-- Initial Password -->
```

Comment:
this should only be changeable on creation. If the user is created, this field should not be updateable. The server accepts this field, but this is a bug. Please add `'default_password'` to the `read_only_fields` in the `UserFullSerializer`.

Reply:
This introduces another bug. The default password cannot be set anymore, rather, the server will always generate a one.